content/docs/reference/types/object.mdx

```diff
@@ -1,12 +1,16 @@
 ---
 title: The "object" field
-last_edited: '2024-09-12T01:50:34.111Z'
+last_edited: '2024-09-26T06:54:19.563Z'
 next: content/docs/reference/types/rich-text.mdx
 previous: content/docs/reference/types/reference.mdx
 ---
 
 ## `object` Type
 
+The `object` field is a dynamic field type creates a nested page in the CMS with the data it contains.
+
+This can be used to create repeated data structures, group related data in the CMS and/or give users the option to choose between a set of templates.
```

Comment:
what are repeated data structures?

Reply:
Like a list of some object (ex. I want a bunch of author objects, with name, books etc.) – author being the data structure

Reply:
lgtm